src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.TargetingPackResolution.targets

```diff
@@ -134,16 +134,15 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <GetPackageDirectory
       Items="@(TargetingPack)"
-      ProjectPath="$(MSBuildProjectFullPath)"
       PackageFolders="@(AssetsFilePackageFolder)">
 
       <Output TaskParameter="Output" ItemName="ResolvedTargetingPack" />
       
     </GetPackageDirectory>    
     
     <ItemGroup>
-      <ResolvedTargetingPack>
-        <Path Condition="'%(ResolvedTargetingPack.Path)' == ''">%(PackageDirectory)</Path>
+      <ResolvedTargetingPack Condition="'%(ResolvedTargetingPack.Path)' == '' and '%(ResolvedTargetingPack.PackageDirectory)' != ''">
```

Comment:
Are you sure that this works correctly?  I'm used to having to put conditions involving custom item metadata on a metadata subelement.  Maybe it's different because this is in a target.

Reply:
cc @rainersigwald 

I'm observing it work correctly. If this is wrong, we have a test hole that we need to fix.